sqlalchemy-stubs/sql/schema.pyi

```diff
@@ -265,7 +266,7 @@ class ForeignKey(DialectKWArgs, SchemaItem):
     match: Optional[str] = ...
     def __init__(
         self,
-        column: Union[Column[Any], str],
+        column: Union[Column[Any], Mapped[Any], str],
```

Comment:
this is ORM, this can't be in the SQL package

Reply:
same this can be any